### PR TITLE
wishlist-page

### PR DESCRIPTION
**Pull request by anshtripathi01**
**_Please review my wishlist page_**
wishlist

### CODE REVIEW

File: Components/Wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    margin: 0.2rem 0 0 1.5rem;
+}
+
+.product-title {
+    margin: 0.1rem 0 0 1.5rem;
+
+}
+
+.like-btn {
+    color: #ff0000;
```
Comment (yashpatel1797) [inline Components/Wishlist/wishlist.css:33]: use var here if you have used everywhere

**Review by yashpatel1797 — APPROVED** [review]
Looks good.

**Review by zabihshaik — APPROVED** [review]
Looks good to merge.

File: Components/Wishlist/wishlist.css (hunk 1 above)
Comment (anshtripathi01) [reply Components/Wishlist/wishlist.css:33]: okay

**Pull request merged**